.circleci/config.yml

```diff
@@ -63,10 +68,15 @@ references:
   store_test_results: &store_test_results
     store_test_results:
       path: ~/rspec
+  save_runtime_log_cache: &save_runtime_log_cache
+    save_cache:
+      key: t1-runtime-log-{{ .Environment.CIRCLE_JOB }}
+      paths:
+        - ~/runtime_rspec.log
   run_rspec: &run_rspec
     run:
       name: Run RSpec
-      command: mkdir ~/rspec && cd decidim-$CIRCLE_JOB && bundle exec rake
+      command: mkdir ~/rspec && cd decidim-$CIRCLE_JOB && bundle exec parallel_test spec/ -n $NUM_CPUS -t rspec --runtime-log ~/runtime_rspec.log -o "--format progress --format ParallelTests::RSpec::RuntimeLogger --out ~/runtime_rspec.log"
```

Comment:
Is this line correctly formatted? Specially the part where it says `-o "--format progress --format ParallelTests::RSpec::RuntimeLogger --out ~/runtime_rspec.log"`

Why is this last part inside a String?

Reply:
Maybe something like this? http://patrobinson.github.io/2015/03/12/multiple-format-outputs-rspec/

Reply:
@rbngzlv ping